static/src/javascripts/projects/common/modules/identity/upsell/account-creation/AccountCreationForm.js

```diff
@@ -20,29 +20,34 @@ class AccountCreationForm extends Component {
             errors: [],
         });
 
-        reqwest({
-            url: '/password/guest',
+        const url = '/password/guest';
+
+        const data = new URLSearchParams();
+        data.append('csrfToken', this.props.csrfToken);
+        data.append('token', this.props.accountToken);
+        data.append('password', this.state.password);
+
+        fetch(url, {
             method: 'post',
```

Comment:
does fetch method need to be uppercase?

Reply:
I think you’re right, but if I [understand the spec correctly][1], it will be normalised to `POST` automatically?

[1]: https://fetch.spec.whatwg.org/#concept-method

```suggestion
            method: 'POST',
```